fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailActivity.java

```diff
@@ -129,9 +129,25 @@ private void openIdentityMenu() {
         () -> IdentityFragment.newInstance(mViewModel.getPublicKey()));
   }
 
-  private void openDetailMenu() {
+  private void openWitnessMenu() {
     setCurrentFragment(
-        getSupportFragmentManager(), R.id.fragment_lao_detail, LaoDetailFragment::newInstance);
+        getSupportFragmentManager(), R.id.fragment_witnessing, WitnessingFragment::newInstance);
+  }
+
+  private void openDigitalCashMenu() {
+    startActivity(
+        DigitalCashActivity.newIntent(
+            this,
+            mViewModel.getCurrentLaoValue().getId(),
+            mViewModel.getCurrentLaoValue().getName()));
```

Comment:
We should change it in a future PR if we have time : it's totally useless to provide the name as well